Command.md

```diff
@@ -0,0 +1,309 @@
+  metadata:
+   name: kubi-svc.kube-system
+  spec:
+    groups:
+      - system:authenticated
+    request: $(cat server.csr | base64 | tr -d '\n')
+    signerName: kubernetes.io/kube-apiserver-client
+    usages:
+      - digital signature
+      - key encipherment
+      - server auth
+  EOF
+  ```
+  
+  - Approving the certificate signing request
+  ```
+  kubectl certificate approve kubi-svc.kube-system
+  kubectl get csr
+  ```
+ 
+  - Create a Certificate Authority
+  ```
+  cat <<EOF | cfssl gencert -initca - | cfssljson -bare ca
+      {
+       "CN": "kube-kubi",
+       "key": {
+       "algo": "rsa",
+       "size": 2048
+        }
+      }
+  EOF
+ 
+  echo '{
+     "signing": {
+          "default": {
+             "usages": [
+                "digital signature",
+                "key encipherment",
+                "server auth"
+            ],
+            "expiry": "876000h",
+            "ca_constraint": {
+                "is_ca": false
+             }
+         }
+      }
+  }' > server-signing-config.json 
+
+  ```
+  - Use a server-signing-config.json signing configuration and the certificate authority key file and certificate to sign the certificate request:
+  ```
+  kubectl get csr kubi-svc.kube-system -o jsonpath='{.spec.request}' | \
+  base64 --decode | \
+  cfssl sign -ca ca.pem -ca-key ca-key.pem -config server-signing-config.json - | \
+  cfssljson -bare ca-signed-server
+  ```
+
+  - Upload the signed certificate
+  ```
+  kubectl get csr kubi-svc.kube-system -o json | \
+  jq '.status.certificate = "'$(base64 ca-signed-server.pem | tr -d '\n')'"' | \
+  kubectl replace --raw /apis/certificates.k8s.io/v1/certificatesigningrequests/kubi-svc.kube-system/status -f -
+  ```
+
+  - Download the certificate and use it
+  ```
+  kubectl get csr kubi-svc.kube-system -o jsonpath='{.status.certificate}' \
+  | base64 --decode > server.crt
+  cat server.crt  
+  ```
+
+  - Create a secret for the deployment
+  ``` 
+  kubectl -n kube-system create secret tls kubi --key server-key.pem --cert server.crt
+  kubectl -n kube-system create secret generic kubi-encryption-secret --from-file=/tmp/ecdsa-key.pem --from-file=/tmp/ecdsa-public.pem
+  kubectl -n kube-system create secret generic kubi-secret  --from-literal ldap_passwd='password'
+  ```
+
+  - Deploy manifest (CRD, prerequisites,local-config) of kubi
+  ```
+  kubectl apply -f https://raw.githubusercontent.com/ca-gip/kubi/master/deployments/kube-deployment.yml
+  kubectl apply -f https://raw.githubusercontent.com/ca-gip/kubi/master/deployments/kube-crds.yml
+  kubectl apply -f https://raw.githubusercontent.com/ca-gip/kubi/master/deployments/kube-prerequisites.yml
+  kubectl apply -f https://raw.githubusercontent.com/ca-gip/kubi/master/deployments/kube-local-config.yml
+  ```
+  
+
+ Copy the secret from you Kubernetes cluster
+ 
+- Create secret dirs on your local machine
+ ```
+ sudo mkdir -p /var/run/secrets/{certs,ecdsa,kubernetes.io}
```

Comment:
Why another folder on the host? Please adapt to a temp folder, or a folder inside this git repo, but gitignored?